main.py

```diff
@@ -62,8 +62,47 @@ def h_run(args):
 		print("Failed to start: No halibot configuration found in the current directory!")
 		return
 
-	logging.basicConfig(level=logging.DEBUG)
 	bot = halibot.Halibot()
+	bot._load_config()
+
+	logfile = None
+	loglevel = logging.DEBUG
+	loglevel_str = "DEBUG"
+	loglevels = { 
+		"CRITICAL": logging.CRITICAL, 
+		"ERROR": logging.ERROR, 
+		"WARNING": logging.WARNING, 
+		"INFO": logging.INFO,
+		"DEBUG": logging.DEBUG, 
+		"NOTSET": logging.NOTSET
+	}
+
+	# Set log file if argument is provided
+	if args.log_file:
+		logfile = args.log_file
+	elif "log-file" in bot.config:
+		logfile = bot.config["log-file"]
+
+	# Set log level if argument is provided
+	if args.log_level:
+		loglevel_str = args.log_level
+	elif "log-level" in bot.config:
+		loglevel_str = bot.config["log-level"]
+
+	# Error check log level string
+	if loglevel_str not in loglevels:
+		print("Log level {} invalid. Continuing with log level {}".format(loglevel_str, loglevel))
```

Comment:
@sjrct Thoughts on this? I'm starting to think that this should be an error, since it implies misconfiguration. Rather than have it limp along, should it just give up and yell at you?